octoprint_psucontrol_tasmota/__init__.py

```diff
@@ -101,6 +101,24 @@ def turn_psu_off(self):
 
 
     def get_psu_state(self):
+        # First try the "Power"-cmd
+        cmd = "Power{}".format(self.config['plug'])
+
+        response = self.send(cmd)
+        if not response:
+            return False
+        data = response.json()
+
+        status = None
+        try:
+            status = (data['POWER'] == 'ON')

```

Comment:
This does not work on tasmota devices with more than 1 relay. If you look at the code pulling the status from the StatusSTS structure it first looks for a key "POWERn" (when n is the plug number set in the plugin settings) and then for "POWER" (but only if the plug setting is 1).
This should work exactly the same when checking the state using the Power command. So your code should be updated to look like this
```python
try:
    status = (data['POWER' + str(self.config['plug'])] == 'ON')
except KeyError:
    pass

if status == None and self.config['plug'] == 1:
    try:
        status = (data['POWER'] == 'ON')
    except KeyError:
        pass
```
For your reference here is the raw output of checking the state with the Power1 command on a plug with 2 relays:
![afbeelding](https://github.com/kantlivelong/OctoPrint-PSUControl-Tasmota/assets/7568570/49abdc62-96d4-423e-8c4b-a98e3d29627b)